### PR TITLE
improve china account template

### LINKED ISSUE #14999: [master][9.0][10.0] account type for some account template of china is wrong.
Impacted versions:
9.0, 10.0, master 

Steps to reproduce:
like  `1401 材料采购` is on the side of  `assets`,  but this is the wrong type as `expense`  in both l10n_cn_standard and l10n_cn_small_business.

Current behavior:
wrong account type

Expected behavior:
need to correct all the wrong account type/

Video/Screenshot link (optional):

### PR DESCRIPTION
**Pull request by jeffery9**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

try to fix #14999 
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cn_standard/data/account_chart_template_data.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+    Copyright (C) 2008-2008 凯源吕鑫 lvxin@gmail.com   <basic chart data>
+    Copyright (C) 2008-now  开阖软件 jeff@osbzr.com    < PM and LTS >
+    Copyright (C) 2017-now  jeffery9@gmail.com
```
Comment (mart-e) [inline addons/l10n_cn_standard/data/account_chart_template_data.xml]: please avoid putting copyright notice in every file, you can put it in the description of the module if needed

File: addons/l10n_cn_standard/__manifest__.py [hunk 2]
```diff
@@ -14,17 +14,21 @@
     'maintainer': 'jeff@osbzr.com',
     'website': 'http://openerp-china.org',
     'description': """
+包含企业会计准则以下数据
```
Comment (mart-e) [inline addons/l10n_cn_standard/__manifest__.py]: Could you also add a description in English (can have both)? This way, non-speakers can understand what this module is about.

**Comment by mart-e** [comment]
Hello,

Thank you for the big changes. However, I am a bit afraid this is too risky for a stable version.
Could you detail what you have changed exactly? Is it only moving from one file to several ones? Is there some records that have been removed?

File: addons/l10n_cn_standard/__manifest__.py (hunk 2 above)
Comment (jeffery9) [inline addons/l10n_cn_standard/__manifest__.py]: nice advice, I should rewrite those message

File: addons/l10n_cn_standard/data/account_chart_template_data.xml (hunk 1 above)
Comment (jeffery9) [inline addons/l10n_cn_standard/data/account_chart_template_data.xml]: ok,

**Comment by jeffery9** [comment]
review CSV file  online  https://topuptw-my.sharepoint.com/personal/jeffery_topuptw_com/_layouts/15/guestaccess.aspx?guestaccesstoken=N18XtqHii1%2b9NAnLGzKdYBLY9nO5qDZMbSLsn7MHNQs%3d&docid=11c72e20cf94147929d2ede6e3fc07784&rev=1

File: addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml [hunk 3]
```diff
@@ -1,794 +1,24 @@
...
+            <field name="reconcile" eval="True" />
+            <field name="user_type_id" ref="account.data_account_type_current_assets" />
         </record>
-
         <!-- Chart template -->
-        <record id="l10n_chart_china_standard_business" model="account.chart.template">
-            <field name="name">中国会计科目表  （财会[2006]3号《企业会计准则》</field>
-            <field name="code_digits" eval="4"/>
-            <field name="currency_id" ref="base.CNY"/>
+        <record id="l10n_cn_standard" model="account.chart.template">
```
Comment (mart-e) [inline addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: Changing xmlid of such objects is not possible in stable version. Please keep the previous one or I will must merge it in master only.

Comment (jeffery9) [inline addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: ok.

Comment (jeffery9) [inline addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: have revert xmlid to origin

**Comment by mart-e** [comment]
Thanks for the changes

**Pull request merged**